testing/cockroach/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Joe Holden <jwh@zorins.us>
+# Maintainer: Joe Holden <jwh@zorins.us>
+pkgname=cockroach
+pkgver=2.1.5
+_pkgver=v2.1.5
+pkgrel=1
+pkgdesc="The open source, cloud-native SQL database."
+url="https://cockroachlabs.com"
+arch="x86_64"
+license="APL2"
+makedepends="go bash cmake autoconf linux-headers ncurses-dev"
+install="$pkgname.pre-install"
+pkgusers="$pkgname"
+pkggroups="$pkgname"
+subpackages="$pkgname-openrc"
+options="!check"
+source="${pkgname}-${pkgver}.tar.gz::https://binaries.cockroachdb.com/${pkgname}-${_pkgver}.src.tgz
+        cockroach.initd
+        cockroach.confd
+	01_no_githooks.patch"
+
+prepare() {
+	cd ${srcdir}/${pkgname}-${_pkgver}
+	patch -p1 -N -i ${srcdir}/01_no_githooks.patch
```

Comment:
Drop these two lines and replace it with:

```
prepare(){
	default_prepare
}
```

Use the builddir variable instead of `cd ${srcdir}/${pkgname}-${_pkgver}` - https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#builddir

Reply:
Theres a commit missing here, prepare() was previously a mess... I was going to leave it until the new release is out but it doesn't look like they're as close as I though they were - I'll push the correct commit in a bit.

Reply:
Was half asleep when I wrote my comment. You can drop prepare(), it is run by default and only explicitly needed when you need to do some extra stuff.